tests/data/dirichlet_data.py

```diff
@@ -233,13 +233,16 @@ def geodesic_shape_test_data(self):
                 space=self.Space(2),
                 n_points=1,
                 time=0.5,
-                expected=(2,),
+                expected=(
+                    1,
+                    2,
+                ),
             ),
             dict(
                 space=self.Space(3),
                 n_points=4,
                 time=0.5,
-                expected=(4, 3),
+                expected=(4, 1, 3),
```

Comment:
It is like this everywhere in the package. And I would say this is the only consistent solution, because not having time as an axis will create ambiguity. e.g. `(4, 3)` means 4 points at a given time or 4 times for a given point?